tests/core/pyspec/eth2spec/test/merge/genesis/test_initialization.py

```diff
@@ -120,3 +120,51 @@ def test_initialize_post_transition(spec):
     assert spec.is_merge_complete(state)
 
     yield 'state', state
+
+
+@with_merge_and_later
+@spec_test
+@single_phase
+@with_presets([MINIMAL], reason="too slow")
+def test_genesis_block_execution_payload(spec):
```

Comment:
This test case should be moved to `tests/core/pyspec/eth2spec/test/merge/unittests/test_genesis.py` since it doesn't create test vectors.